### PR TITLE
Web3Auth WebGL Window Popup

### PR DESCRIPTION
**Pull request by kantagara**
Finally WebGL builds won't suffer from re-loading the same window over and over again, but the login process will be done in a completely separate window.

### CODE REVIEW

File: Packages/io.chainsafe.web3-unity.ramp/Runtime/Scripts/RampExchangerUniversal.cs [hunk 1]
```diff
@@ -9,20 +10,28 @@ public class RampExchangerUniversal : IRampExchanger, ILifecycleParticipant
...
+        public RampExchangerUniversal(IRampExchangerConfig config, ISigner signer, IAnalyticsClient analyticsClient)
         {
             this.signer = signer;
             this.config = config;
+            this.analyticsClient = analyticsClient;
         }
 
         public ValueTask WillStartAsync()
         {
+            analyticsClient.CaptureEvent(new AnalyticsEvent()
```
Comment (kantagara) [inline Packages/io.chainsafe.web3-unity.ramp/Runtime/Scripts/RampExchangerUniversal.cs:30]: Sneaked this one in, forgot we didn't have analytics in Ramp and Jay needed it 😢

File: Packages/io.chainsafe.web3-unity.web3auth/Runtime/Web3AuthWallet.cs [hunk 2]
```diff
@@ -134,7 +136,6 @@ private TWeb3Auth CreateCoreInstance()
             Object.DontDestroyOnLoad(gameObject);
 
             var instance = gameObject.GetComponent<TWeb3Auth>();
-            instance.Awake();
```
Comment (kantagara) [inline Packages/io.chainsafe.web3-unity.web3auth/Runtime/Web3AuthWallet.cs:137]: This was completely unnecessary since awake is already being called.

File: Packages/io.chainsafe.web3-unity.ramp/Runtime/Scripts/RampExchangerUniversal.cs (hunk 1 above)
Comment (sneakzttv) [reply Packages/io.chainsafe.web3-unity.ramp/Runtime/Scripts/RampExchangerUniversal.cs:30]: noice

File: src/UnitySampleProject/Assets/Samples/web3.unity SDK Web3Auth/2.5.1/Web3.Unity Web3Auth Samples/Scripts/Web3AuthLoginProvider.cs [hunk 3]
```diff
@@ -47,55 +62,58 @@ public void SetRememberMe(bool rememberMe)
...
-        {
-            useProvider = false;
-
-            await TryLogin();
-           
-        }
-#else
+        #if !UNITY_EDITOR && UNITY_WEBGL
+        Web3AuthWebGLConnected += Web3AuthSet;
+        var projectSettings = ProjectConfigUtilities.Load();
```
Comment (oleksandrchainsafe) [inline src/UnitySampleProject/Assets/Samples/web3.unity SDK Web3Auth/2.5.1/Web3.Unity Web3Auth Samples/Scripts/Web3AuthLoginProvider.cs:69]: You should use the IChainConfig that is registered in DI instead, as a user can provide a different config than the one returned by ProjectConfigUtilities.Load().

**Review by oleksandrchainsafe — CHANGES_REQUESTED** [review]
Haven't look much into the JS code as it gives me headache. Overall looks good to me, just one issue has to be fixed.

**Review by sneakzttv — APPROVED** [review]

File: src/UnitySampleProject/ProjectSettings/ProjectSettings.asset
Comment (rob1997) [inline src/UnitySampleProject/ProjectSettings/ProjectSettings.asset:1]: are these changes necessary?

Comment (rob1997) [reply src/UnitySampleProject/ProjectSettings/ProjectSettings.asset:1]: if they're inconsequential feel free to resolve this

Comment (kantagara) [reply src/UnitySampleProject/ProjectSettings/ProjectSettings.asset:1]: I mean it's nice to have package to it's latest version. Didn't wanna burden us with another PR just for that.
